src/Components/MyListsHandler.cs

```diff
@@ -25,192 +27,206 @@ public class MyListsHandler
 
-                // Add delete button to the header panel
-                headerPanel.Children.Add(deleteButton);
+            int textFont = 18;
 
-                // Create the Expander for the list
-                Expander listExpander = new Expander
-                {
-                    Header = headerPanel,
-                    FontSize = 18,
-                    Foreground = textColor,
-                    Margin = new Thickness(0, 10, 0, 10)
-                };
+            // Create Popup window
+            Window popup = new Window
+            {
+                Title = "Deleting " + listName,
+                Width = 500,
+                Height = 200,
+                WindowStartupLocation = WindowStartupLocation.CenterScreen,
+                ResizeMode = ResizeMode.NoResize,
+                Background = background
+            };
 
-                StackPanel ingredientPanel = new StackPanel();
+            // Panel for the popup content
+            StackPanel panel = new StackPanel { Margin = new Thickness(10) };
 
-                // Add search box for filtering ingredients
-                TextBox searchBox = new TextBox
-                {
-                    Text = "Search ingredients...", // Initial placeholder text
-                    Foreground = Brushes.Gray,
-                    Margin = new Thickness(0, 0, 0, 10)
-                };
+            // Confirmation message text
+            TextBlock confirmationText = new TextBlock
+            {
+                Text = "Are you sure you want to delete " + listName + "?",
+                Foreground = textColor,
+                FontSize = textFont,
+                FontWeight = FontWeights.Bold,
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Top,
+                Margin = new Thickness(10)
+            };
 
-                // Placeholder behavior
-                searchBox.GotFocus += (s, e) =>
-                {
-                    if (searchBox.Text == "Search ingredients...")
-                    {
-                        searchBox.Text = "";
-                        searchBox.Foreground = Brushes.Black;
-                    }
-                };
+            panel.Children.Add(confirmationText);
 
-                searchBox.LostFocus += (s, e) =>
-                {
-                    if (string.IsNullOrWhiteSpace(searchBox.Text))
-                    {
-                        searchBox.Text = "Search ingredients...";
-                        searchBox.Foreground = Brushes.Gray;
-                    }
-                };
+            // Create the delete confirmation button 
+            Button deleteButton = new Button
+            {
+                Content = "Delete",
+                Style = (Style)Application.Current.Resources["ExpandButtonStyle"], 
+                Margin = new Thickness(10, 30, 10, 10)
```

Comment:
Could you make delete buttons red text colour to indicate it's a negative action.
![image](https://github.com/user-attachments/assets/37f06273-c794-418f-baaa-70bc9f708bc4)